x/gov/keeper/tally.go

```diff
@@ -40,25 +40,25 @@ func (keeper Keeper) Tally(ctx sdk.Context, proposal types.Proposal) (passes boo
 		if val, ok := currValidators[valAddrStr]; ok {
 			val.Vote = vote.Option
 			currValidators[valAddrStr] = val
-		} else {
-			// iterate over all delegations from voter, deduct from any delegated-to validators
-			keeper.sk.IterateDelegations(ctx, vote.Voter, func(index int64, delegation exported.DelegationI) (stop bool) {
-				valAddrStr := delegation.GetValidatorAddr().String()
+		}
 
-				if val, ok := currValidators[valAddrStr]; ok {
-					val.DelegatorDeductions = val.DelegatorDeductions.Add(delegation.GetShares())
-					currValidators[valAddrStr] = val
+		// iterate over all delegations from voter, deduct from any delegated-to validators
```

Comment:
_I think_ this makes sense. I would like to see some documentation added to this comment for this specific case.